docs/integrations/app-development/opentelemetry/jfrog-artifactory-opentelemetry.md

```diff
@@ -0,0 +1,175 @@
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Artifactory-OpenTelemetry/Artifactory-Schematics.png')} alt="Artifactory-Schematics" />
+
+## Fields creation in Sumo Logic for Artifactory
+
+Following are the Tags which will be created as part of Artifactory App install if not already present.
+
+**sumo.datasource** - Has fixed value of **artifactory**
+
+## Prerequisite
+
+This section provides instructions for configuring log collection for Artifactory for the Sumo Logic App.
+
+The Sumo Logic App for Artifactory collects data from the following logs:
+
+-   **artifactory.log** : The main Artifactory log file that contains data on Artifactory server activity.
+-   **access.log** :  The security log containing important information about accepted and denied requests, configuration changes, and password reset requests. The originating IP address for each event is also recorded.
+-   **request.log** :  Generic HTTP traffic information similar to the Apache HTTPd request log.
+-   **traffic.log** :  A log that contains information about site traffic and file sizes.
+
+For more details about Artifactory logs, refer to <https://www.jfrog.com/confluence/display/RTF/Artifactory+Log+Files> and [Artifactory Log Files](https://www.jfrog.com/confluence/display/RTF6X/Artifactory+Log+Files#ArtifactoryLogFiles-RequestLog).
+
+Sumo Logic reads logs in the directory `/var/opt/jfrog/artifactory/logs`:
+
+-   `artifactory.log`
+-   `access.log`
+-   `request.log`
+-   `traffic.*.log`
+
+To activate the `traffic.log` file, add the following parameter to your `artifactory.system.properties` file, located under `$ARTIFACTORY/etc` :
+
+artifactory.traffic.collectionActive=true
+
+A restart is required for traffic collection to take effect.
+
+## Collection Configuration & App installation
+
+As part of the setting up the collection process and app installation user can select the App from App Catalog and click on Install App. Please follow the steps below:
+
+### Step1: Set up Collector
+
+If you want to use an existing Otel Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Artifactory-OpenTelemetry/Artifactory-Collector.png')} alt="Artifactory-Collector" />
```

Comment:
According to our [style guide](https://help.sumologic.com/docs/contributing/style-guide/#images), "Images must be added to the static/img folders." Is it acceptable to point to an image outside of our file structure?